nl_server/custom_dc_env.yaml

```diff
@@ -0,0 +1,10 @@
+# TODO: move this to deploy folder. This is here for the easy of
+# custom DC loading for now.
+
+default_indexes:
```

Comment:
Can you add a short comment for the 3 properties here - `default_indexes`, `enabled_indexes`, `enable_reranking`?